astropy/utils/console.py

```diff
@@ -498,7 +487,7 @@ def human_file_size(size):
     return "{0:>3s}{1}".format(str_value, suffix)
 
 
-class ProgressBar(six.Iterator):
+class ProgressBar(object):
```

Comment:
That seems odd.

Reply:
I think that's me (?). I remember looking up the doc and it says `six.Iterator` is just an alias to `object` in Python 3.

Reply:
It's for the `__next__` method : https://pythonhosted.org/six/#six.Iterator